driver-core/src/test/unit/com/mongodb/internal/connection/ClientMetadataHelperSpecification.groovy

```diff
@@ -63,66 +68,10 @@ class ClientMetadataHelperSpecification extends Specification {
     def 'should create client metadata document including driver info'() {
         given:
         def applicationName = 'appName'
-        def driverInfo = MongoDriverInformation.builder().driverName('mongo-spark').driverVersion('2.0.0')
-                        .driverPlatform('Scala 2.10 / Spark 2.0.0').build()
-
-        expect:
-        createClientMetadataDocument(applicationName, driverInfo) == createExpectedClientMetadataDocument(applicationName, driverInfo)
-    }
-
-    def 'should create client metadata document and exclude the extra driver info if its too verbose'() {
```

Comment:
These tests have been moved (altered where needed) to the java prose test.

Reply:
Does this imply converting the groovy file to java?

I'm unsure about unit vs prose. Perhaps all of these tests should try to send `hello` at the end? The existing tests seemed to me very similar - like tests that should have been prose tests, but weren't specified whenever this was originally created.

Reply:
Yes, I would ditch any / all groovy spec tests. 

I suppose sending `hello` isn't required to test the what this class generates which in my mind makes it more of a unit test.  However, I have no issue with having them mixed in with prose tests and all being consistent.

Reply:
The issue with sending `hello` is it should be done sync and async - if the output is tested against.